portal/migrations/versions/c49b7f7944a7_.py

```diff
@@ -0,0 +1,64 @@
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import sessionmaker
+
+from portal.models.audit import Audit
+from portal.models.user_consent import UserConsent
+
+"""consent audit & acceptance_date cleanup
+
+Revision ID: c49b7f7944a7
+Revises: 0f1576a4e220
+Create Date: 2018-06-21 14:37:20.145030
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'c49b7f7944a7'
+down_revision = '0f1576a4e220'
+Session = sessionmaker()
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user_consents', sa.Column(
+        'acceptance_date', sa.DateTime(), nullable=True))
```

Comment:
Should this be `acceptance_datetime`?

Reply:
it's mirroring the string used in all the consent APIs - pretty significant change if you want that...

Reply:
Fair enough :)